homeassistant/components/api/__init__.py

```diff
@@ -195,6 +196,7 @@ async def get(self, request):
             # always needs authentication
             ATTR_REQUIRES_API_PASSWORD: True,
             ATTR_VERSION: __version__,
+            ATTR_CURRENCY: None,
```

Comment:
Do you remember why this was added? It's never set nor does it seem to be something we should make part of discovery info.